src/build/index.js

```diff
@@ -14,11 +14,12 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-const buildSVGs = require('./buildSVGs');
-const buildTool = require('./buildTool');
-const cleanDist = require('./cleanDist');
-const createGitHubComment = require('./createGitHubComment');
-const ensureDir = require('./ensureDir');
-const fetchTemplate = require('./fetchTemplate');
+const buildSVGs = source => require('./buildSVGs')(source);
+const buildTool = (source, out) => require('./buildTool')(source, out);
+const cleanDist = () => require('./cleanDist')();
+const createGitHubComment = args => require('./createGitHubComment')(args);
+const ensureDir = dir => require('./ensureDir')(dir);
+const fetchTemplate = () => require('./fetchTemplate')();
+const dev = (source, out, port) => require('./dev')(source, out, Number(port));
```

Comment:
Why? We seem to be wrapping this is anon functions that are just passing the args through, which seems rather redundant?

Only diff being the `Number` wrap on port, which we can do directly in `cli.js`?

Reply:
This is to make debugging easier. If we import them all at once, a lot of require noise gets pushed in for every command, meaning one command that would normally perfectly run can error due to another file.

Reply:
I wonder if we can actually just remove this file completely? It seems that everything here is also a 1:1 map with file names, which we can just directly import where needed?